buildroot-external/rootfs-overlay/etc/NetworkManager/NetworkManager.conf

```
@@ -5,7 +5,7 @@ autoconnect-retries-default=0
 rc-manager=file
 
 [keyfile]
-unmanaged-devices=type:bridge;type:tun;driver:veth
+unmanaged-devices=type:bridge;type:tun;driver:veth;except:interface-name:bridge*
```

So with this bridge named devices are allowed.

Does removing `type:bridge` cause problems for the docker bridge? Maybe doing the reverse makes more sense: Remove `type:bridge` and add the interface named `hassio` and `docker*` to the unmanaged list.

---

I don't know the reason why all bridge interfaces have been added to the unmanaged list. So I've provided the a minimal change, that shouldn't break anything.

---

Ok, but this means you need to name the bridge something starting with `bridge..`. Examples often use `br0`, which would not work now right?

I only see Docker as the reason why it has been set as unmanaged. But I'd say its better to reverse the logic now, so:

```suggestion
unmanaged-devices=type:tun;driver:veth;interface-name:docker*;interface-name:hassio
```

---

Do you have any other objections against this PR?